src/npc/npc.gd

```diff
@@ -0,0 +1,31 @@
+extends Area2D
```

Comment:
we should have chest use the same scene as other NPC, as they're all functionally the same.